docs/source/markdown/options/network.md

```diff
@@ -15,17 +15,23 @@ Valid _mode_ values are:
   - **mac=MAC**: Specify a static mac address for this container.
   - **interface_name**: Specify a name for the created network interface inside the container.
 
-  For example, to set a static ipv4 address and a static mac address, use `--network bridge:ip=10.88.0.10,mac=44:33:22:11:00:99`.
-
-- \<network name or ID\>[:OPTIONS,...]: Connect to a user-defined network; this is the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the bridge network mode. It is possible to specify the same options described under the bridge mode above. Use the **--network** option multiple times to specify additional networks.
+    For example, to set a static ipv4 address and a static mac address, use `--network bridge:ip=10.88.0.10,mac=44:33:22:11:00:99`.
 
+- **\<network name or ID\>[:OPTIONS,...]**: Connect to a user-defined network; this is the network name or ID from a network created by **[podman network create](podman-network-create.1.md)**. Using the network name implies the bridge network mode. It is possible to specify the same options described under the bridge mode above. Use the **--network** option multiple times to specify additional networks. \
   For backwards compatibility it is also possible to specify comma-separated networks on the first **--network** argument, however this prevents you from using the options described under the bridge section above.
```

Comment:
Doesn't this just remove the new paragraph? IMO the extra paragraph makes it easier to read.

Reply:
Well, I couldn't get new paragraphs to work the way they should. If it's a separate paragraph, it seems to indent it way less than the list itself. Lists seem to be buggy anyway with gomd2man, they are all indented a step unnecessarily.

Reply:
I agree that it would be nicer to replace all forced newlines (`\` at eol) with new paragraphs but I don't think there's a way to do it right now, `gomd2man` seems to merge them into a single paragraph for some reason. (btw, couldn't sphinx generate man pages as well? not that it would necessarily solve this)

Reply:
Here's why: https://github.com/cpuguy83/go-md2man/blob/master/md2man/roff.go#L144-L147

I'm not sure if that comment is correct though.

Reply:
I am fine with this change if it breaks man pages. I have slight preference for the paragraph but if the generation cannot handle it today then it is fine.

Reply:
Apparently it's a pretty simple fix, check out https://github.com/baltitenger/podman/tree/man-network-fix-2. There I also patched the man page generator to handle paragraphs in list items. I'm not sure if it would have an impact anywhere else, would be best to check the diff before/after. I guess this change should be separated from the docs changes as well.

Reply:
I have submitted a pr to cpuguy83/go-md2man#116. Not sure how active that project is, so I guess in the meantime this pr could be merged as is and updated once go-md2man has a release with this merged. Or you can patch the module, idk.

Reply:
I rather not use a patched go-md2man here, agreed for now your changes are good, thanks!